adapter-tcp/src/main/java/io/nosqlbench/adapter/tcpclient/TcpClientOpDispenser.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022 nosqlbench
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.nosqlbench.adapter.tcpclient;
+
+import io.nosqlbench.engine.api.activityimpl.BaseOpDispenser;
+import io.nosqlbench.engine.api.templating.ParsedOp;
+
+import java.util.function.LongFunction;
+
+public class TcpClientOpDispenser extends BaseOpDispenser<TcpClientOp, TcpClientAdapterSpace> {
+
+    private final LongFunction<TcpClientAdapterSpace> ctxFunction;
+    private final LongFunction<String> outFunction;
+
+    public TcpClientOpDispenser(TcpClientDriverAdapter adapter, ParsedOp cmd, LongFunction<TcpClientAdapterSpace> ctxfunc) {
+        super(adapter,cmd);
+        this.ctxFunction = ctxfunc;
+        LongFunction<Object> objectFunction = cmd.getAsRequiredFunction("stmt", Object.class);
+        LongFunction<String> stringFunction = l -> objectFunction.apply(l).toString();
+        cmd.enhanceFuncOptionally(stringFunction,"suffix",String.class,(a, b) -> a+b);
```

Comment:
This is special to stdout, whereas we now have integrated string templating in the form of "prefix{binding}suffix" format support. We should probably remove this op template field to avoid duplicity or confusion.

Reply:
To make sure I got this right, does this mean we no longer need to add something like:
`cmd.enhanceFuncOptionally(stringFunction,"prefix",String.class,(a, b) -> b+a);`
Because we support string templating to add a prefix to a binding, and that can do that work for us without parsing a special 'prefix' parameter?